src/actions/locationActions.js

```diff
@@ -2,7 +2,7 @@ import {
   SET_LOCATION,
 } from '../constants/actionTypeConstants';
 
-export default function setLocation(locationId) {
+export function setLocation(locationId) {
```

Comment:
why remove the `default`, since there's only one export?  Unless you plan to add in more exports?